pkg/template/template.gen.go

```diff
@@ -31,6 +31,18 @@ import (
 	"istio.io/mixer/pkg/expr"
```

Comment:
The generated file is copied to this dir via a script. 

In the future this is what we should be doing when we have the Build that would create a build of Mixer from all templates:
The build rule would invoke the tools/codegen/cmd/mixgenbootstrap to generate the template.gen.go... create a go_library out of it and make the mixer/pkg/template reference the on the fly generated go_library (which just contains the template.gen.go)